episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+```
+
+Note that this image has a total of 3 dimensions, and so will also be stored in 
+a 3D array:
+
+![](fig/tyx-array.png){alt="A diagram of a tyx image array" width='40%'}
+
+This makes the point that certain dimensions/axes aren't always in the same 
+positions. For example, a 3D image array with shape (512, 512, 512) could 
+represent a zyx, cyx or tyx image. We'll discuss this more in the next section. 
+
+## Dimension order
+
+As we've seen so far, we can check the number and size of an image's dimensions 
+by running:
+```python
+image.shape
+```
+
+Napari reads this array and displays the image appropriately, with the correct 
+number of sliders, based on these dimensions. It's worth noting though that 
+Napari doesn't usually know what these different dimensions represent e.g. 
+consider a 4 dimensional image with shape (512, 512, 512, 512). This could be 
+tcyx, czyx, tzyx etc... Napari will just display it as an image with 2 
+additional sliders, not caring about exactly what each represents.
+
+Python has certain conventions for the order of image axes (like 
+[scikit-image's 'coordinate conventions'
+](https://scikit-image.org/docs/stable/user_guide/numpy_images.html#coordinate-conventions) 
+and [aicsimageio's reader
+](https://allencellmodeling.github.io/aicsimageio/aicsimageio.aics_image.AICSImage.html)) 
+- but this tends to vary based on the library or plugin you're using. They're 
+not firm rules!
+
+Therefore, it's always worth checking you understand which axes are being shown 
+in Napari and what they represent! Check against your prior knowledge of the 
+experimental setup, and check the metadata in the original image (we'll look at 
+this in the next episode). If you want to change how axes are displayed, 
+remember you can use the roll or transpose dimensions buttons as discussed in 
+the [imaging software episode](imaging-software.md). Also, if loading the image 
+manually from the console, you can provide some extra information like the 
+`channel_axis` parameter discussed above. 
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Interpreting image dimensions
+
+Remove all image layers, then open a new image by copying and pasting the 
+following into Napari's console:
+
+```python
+from skimage import data
+image = viewer.add_image(data.kidney(), rgb=False).data
+```
+
+This is a fluorescence microscopy image of mouse kidney tissue. 
+
+1. How many dimensions does it have?
+
+2. What do each of those dimensions represent? (e.g. t, c, z, y, x) **Hint:** 
+try using the roll dimensions button ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/roll.svg
+){alt="A screenshot of Napari's roll dimensions button" height='25px'} to view 
+different combinations of axes.
+
+3. Once you know which dimension represent channels, remove the image and load 
+it again with:
+  ```python
+  # Replace ? with the correct channel axis
+  viewer.add_image(data.kidney(), rgb=False, channel_axis=?)
```

Comment:
If you use channel_axis = 1 or 2 napari will crash (at least mine did). I guess it doesn't like the idea of a 512 channel image. It might be worth adding a reassuring note saying something like "Using the wrong channel axis may cause napari to crash, if this happens to you just restart napari and try again."

Reply:
This may also play into my previous comment about a lot of layers not being very practical, so you if you do address that comment, here you could also say "Remember what we discussed previously about how storing images as layers don't make a lot of sense when it would create a lot of layers" or something to that effect. 
You could even put @thompson318 suggested warning in that discussion as a general FYI Napair (and probably lots of viewers I'm guessing) will crash if you try to load a lot of layers all at once.